astropy/visualization/wcsaxes/core.py

```diff
@@ -323,7 +327,11 @@ def reset_wcs(self, wcs=None, slices=None, transform=None, coord_meta=None):
             # by hand. For example if the user sets a celestial WCS by hand and
             # forgets to set the units, WCS.wcs.set() will do this.
             if wcs is not None:
-                wcs.wcs.set()
+                # Check if the WCS object is an instance of `astropy.wcs.WCS`
+                # This check is necessary as only `astropy.wcs.WCS` supports
+                # wcs.set() method
+                if isinstance(wcs, WCS):
+                    wcs.wcs.set()
```

Comment:
Do we have the corresponding method for `BaseLowLevelWCS`?

Reply:
No, this is specific to FITS-WCS